_ml-commons-plugin/custom-local-models.md

```diff
@@ -20,12 +20,14 @@ As of OpenSearch 2.11, OpenSearch supports local sparse encoding models.
 
 As of OpenSearch 2.12, OpenSearch supports local cross-encoder models.
 
+As of OpenSearch 2.13, OpenSearch supports local question answering models.
+
 Running local models on the CentOS 7 operating system is not supported. Moreover, not all local models can run on all hardware and operating systems.
 {: .important}
 
 ## Preparing a model
 
-For both text embedding and sparse encoding models, you must provide a tokenizer JSON file within the model zip file.
+For all the models, you must provide a tokenizer JSON file within the model zip file.
```

Comment:
ZIP file?

Reply:
I don't think zip file should be capitalized.